src/components/map/Map.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import StyledButton from "../common/CommonStyle";
+
+const Map = ({ map }) => {
+  const navigate = useNavigate();
+
+  return (
+    <MapButton onClick={() => navigate(`${map.id}`)}>{map.title}</MapButton>
+  );
+};
+
+export default Map;
+
+const MapButton = styled(StyledButton)`
+  display: flex;
+  justify-content: center;
+  margin: 10px;
+  width: 60vw;
```

Comment:
```suggestion
  width: 60vw;
  margin: 10px;
```

width margin 순서가 바뀌어야 할 것 같습니다~